Models/ImageClassification/SqueezeNet.swift

```diff
@@ -115,3 +116,68 @@ public struct SqueezeNet: Layer {
         return convolved2
     }
 }
+
+public struct SqueezeNetV1_1: Layer {
+    public var conv1 = Conv2D<Float>(filterShape: (3, 3, 3, 64), strides: (2, 2), padding: .same,
```

Comment:
```suggestion
    public var conv1 = Conv2D<Float>(
        filterShape: (3, 3, 3, 64), 
        strides: (2, 2), 
        padding: .same,
        activation: relu)
```
Please follow the Google Swift Style Guide or run swift-format.

Reply:
swift-format doesn't modify it this way :P, I'll update it as you said.